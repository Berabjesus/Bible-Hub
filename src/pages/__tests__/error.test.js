import React from "react";
import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import Error from "../error";
import { BrowserRouter } from "react-router-dom";

describe("Bible page", () => {
  it("matches the snapshot", () => {
    const component = renderer.create(
      <BrowserRouter>
          <Error />
      </BrowserRouter>
    );
    expect(component).toMatchSnapshot();
  });
});