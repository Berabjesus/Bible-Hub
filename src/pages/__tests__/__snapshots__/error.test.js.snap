// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bible page matches the snapshot 1`] = `
<div
  className="centered"
>
  <h1>
    404: Page Not Found
  </h1>
</div>
`;
